CMP1903_A1_2324/Die.cs

```diff
@@ -6,18 +6,19 @@
 
 namespace CMP1903_A1_2324
 {
-    internal class Die
+    public class Die
     {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
 
-        //Property
-
-
-        //Method
+        private static Random random = new Random();    // This is to hold a single instance of Random
 
+        //Private fields to store values
+        private int currentValue;
+        
 
+        public int Roll()
+        {
+            currentValue = random.Next(1,7); // This is inclusive, gets a number between 1 and 6
+            return currentValue;
+        }
```

Comment:
Good and easy to understand Die.cs program, does as what the requirements expected, returns a statement, generates a random number between 1 - 6.